Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -860,6 +860,23 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(gd:
         .AddCodeOnlyParameter("conditionInverted", "")
         .MarkAsAdvanced();
 
+    extension.AddCondition("Raycast",
+                   _("Raycast"),
+                   _("Sends a ray from the given position and angle, intersecting the first object.\nThe instersected object will become the only one taken into account."),
+                   _("Raycast _PARAM0_ from _PARAM1_;_PARAM2_, and save the result in _PARAM5_, _PARAM6_"),
+                   _("Collision"),
+                   "res/conditions/collision24.png",
+                   "res/conditions/collision.png")
+        .AddParameter("objectList", _("Objects to test"))
```

Comment:
We could have more verbose parameter, to be sure that everyone understand how to use the condition :)

`Objects to test against the ray`
`X position of the source of the ray`
`Y position of the source of the ray`
`Angle of the ray (in degrees)`
`Maximum distance (in pixels)`
`Variable where to store the X position of the intersection`
`Variable where to store the Y position of the intersection`

Tell me if it makes sense for you :)

Reply:
What about:
`Objects to test against the ray`
`Ray source X position`
`Ray source Y position`
`Ray angle (in degrees)`
`Ray maximum distance (in pixels)`
`Variable where to store the X position of the intersection`
`Variable where to store the Y position of the intersection`
?

Reply:
Yes looks consistent and clear! Let's go for it :)